src/service/ampdoc-impl.js

```diff
@@ -300,6 +303,14 @@ export class AmpDoc {
   contains(node) {
     return this.getRootNode().contains(node);
   }
+
+  /**
+   * Binding of 3p analytics vendors' responses, used amp-ad-exit
+   * @return {!Object<string, !Object<string, Object<string,string>>>}
+   */
+  getIframeTransportResponses() {
```

Comment:
I don't fully understand why we have class name as string literal in the response. But if that's really needed, amp.extern.js would be your last try.

if you're talking about sharing data between 2 extensions, `window['a-good-name-for-your-data']` can be a possible solution.